### ISSUE TITLE
@Twitter 12th! but reply chain feature is to difficult to beginner user is me

### ISSUE THREAD

**Issue by YumaInaura** [YumaInaura/YumaInaura#187]
# @Twitter 12th! but reply chain feature is to difficult to beginner user is me

# In  Web UI with Google Chrome

chainged blue line

![image](https://user-images.githubusercontent.com/13635059/50585599-1f659380-0eb9-11e9-90f6-9b0411c40b56.png)

https://twitter.com/sharara1998/status/1080387116599730178

# There is two replay form 

![image](https://user-images.githubusercontent.com/13635059/50585650-618ed500-0eb9-11e9-84b9-a72bbb9983b1.png)

![image](https://user-images.githubusercontent.com/13635059/50585651-65baf280-0eb9-11e9-8db8-d6c0f22ef988.png)

Which shoud i user ...?

# But this URL

not chainged

![image](https://user-images.githubusercontent.com/13635059/50585634-5045c880-0eb9-11e9-9f52-a7f2eed92a6d.png)

https://twitter.com/YumaInaura/status/1080389129869783040

# Two replay form is there

![image](https://user-images.githubusercontent.com/13635059/50585683-90a54680-0eb9-11e9-9621-51f3918070f0.png)
![image](https://user-images.githubusercontent.com/13635059/50585684-913ddd00-0eb9-11e9-97b9-c5406c7e927f.png)

# Which should i use?

- I dont under stand which is like "Proxy" chained replay
- Im thinking abount an hour but i dont understand
- Actually my understanding Twitter skill is too low but i maybe never unstand reply feature with this UI..........

# UX

- I use stand before "No See Form" for replay
- Twitter dear other users feel "Too noisey" my replay
- But im not unerstanding "What is not noisey reply chain" 

![image](https://user-images.githubusercontent.com/13635059/50585770-eda0fc80-0eb9-11e9-8736-f5118a678e33.png)

# TL DR Japanese

- Twitterのリプライの仕組みめちゃくちゃ難しい
- UI UX 的に｢どっちに何をつけたら、どう返信がコンボするの？｣が全然分からないし,2時間考えても直感的に分からないし、何度でも間違う自信がある
- Twitterで｢串方式じゃないリプライ｣をつけて他のユーザーに迷惑をかけてしまったのをUI のせいにするのは日本人的なマナーに反すると思うけどとにかく本当に理解できない‥むずい

---

https://github.com/YumaInaura/YumaInaura/issues
